code-challenges.test.js

```diff
@@ -21,36 +20,161 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe("fibSequence", () => {
+  it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    expect(fibSequence(6)).toEqual([1, 1, 2, 3, 5, 8]);
+    expect(fibSequence(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+  });
+});
+
+// Good failure! Hooray
 
 // b) Create the function that makes the test pass.
 
+// PC:
+// create fn called fibSequence
+// takes in a number
+// made a new array starting at [1, 1] because every number starts that way
+// use iteration with (number - 2) because we start with [1, 1] already
+// .push our array position to the next index
+// rinse, repeat
+// return the array
+
+const fibSequence = (number) => {
+  let myNewArr = [1, 1];
+  // [1, 1, 2, 3, 5, 8 ]
+  for (let i = 0; i < number - 2; i++) {
+    myNewArr.push(myNewArr[i] + myNewArr[i + 1]);
+  }
+  return myNewArr;
+};
+
+console.log(fibSequence(6));
```

Comment:
With the testing suites, you do not need to console.log or do function calls...because it is all taken care of by the test you created...the expect statement does it for you.